scripts/rancher_pr_deploy.sh

```diff
@@ -24,13 +24,15 @@ if [[ -n "$TRAVIS" ]]; then
     curl -s https://raw.githubusercontent.com/nhsuk/nhsuk-rancher-templates/master/templates/c2s-pharmacy-finder/0/docker-compose.yml  -o docker-compose.yml
     curl -s https://raw.githubusercontent.com/nhsuk/nhsuk-rancher-templates/master/templates/c2s-pharmacy-finder/0/rancher-compose.yml -o rancher-compose.yml
 
+    LATEST_RELEASE=$(curl -s https://api.github.com/repos/nhsuk/nearby-services-api/releases/latest | jq -r '.tag_name')
```

Comment:
Note that this change assumes that the pre-release option has been deselected in GitHub for the current release. See [here](https://developer.github.com/v3/repos/releases/#get-the-latest-release) for more details. It also assumes that there is an image associated with the release - I think these have been created manually so far.